runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -243,4 +243,15 @@ default boolean swapAssignment()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		position = 19,
+		keyName = "dagganothKingsLadder",
+		name = "Dagganoth kings",

```

Comment:
We usually name these fields by what they swap, so this one should probably be named just "Slayer" and include what it swaps in description

Reply:
well, I don't know if it will work on other things which is why I just wrote that there